apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ExceptionLoggerTest.cs

```diff
@@ -24,7 +24,7 @@
 using System.Web.Http.ExceptionHandling;
 using Xunit;
 
-namespace Google.Devtools.AspNet.IntegrationTests
+namespace Google.Cloud.Diagnostics.AspNet.IntegrationTests
```

Comment:
(Not related to this line, but I can't add a comment to a file that's only been renamed.)

All the xproj files need changing to change the root namespace. I'd recommend cleaning all the bin/obj directories, then using something like Sublime Text to find any occurrence of "devtools" in all files.